src/c++/library/grpc_client.cc

```diff
@@ -349,6 +350,25 @@ InferResultGrpc::IsNullResponse(bool* is_null_response) const
   return Error::Success;
 }
 
+Error
+InferResultGrpc::Output(std::string& output) const
+{
+  // Only supports LLM outputs with name 'text_output' currently
+  if (output_name_to_buffer_map_.find("text_output") ==
+      output_name_to_buffer_map_.end()) {
+    return Error::Success;
+  }
+
+  const uint8_t* buf{nullptr};
+  size_t byte_size{0};
+  Error e{RawData("text_output", &buf, &byte_size)};
```

Comment:
Under the hood, the C++ grpc/http clients know to grab the `text_output` field. Other clients like the OpenAI client will have to implement this to grab the `choices` -> `message` -> `content` field from here: https://platform.openai.com/docs/api-reference/chat/create

Reply:
I would strongly prefer that we do not take this approach. This only works for one specific OpenAI endpoint.

Reply:
Dumping all outputs now